Library/Formula/cdrtools.rb

```diff
@@ -45,6 +60,10 @@ def install
 
   test do
     system "#{bin}/cdrecord", "-version"
+    system "#{bin}/cdda2wav", "-version"
```

Comment:
Cool. BTW, if this pull request is pushed to master, will Jenkins rebuild everything and refresh the bottles? I hope not, because it would not be required, since the only changes with master are the move of the bottle block and the definition of the test block.